regions.go

```diff
@@ -37,7 +37,6 @@ const (
 	ACLB                   string = "Akamai Cloud Load Balancer"
 	SupportTicketSeverity  string = "Support Ticket Severity"
 	Backups                string = "Backups"
-	PlacementGroup         string = "Placement Group"
 	DiskEncryption         string = "Disk Encryption"
```

Comment:
It looks like these consts might conflict with their struct counterparts. We might want to rename them to something like `CapabilityPlacementGroup`, etc.